### PR TITLE
adding back in read the docs libraries

### PR DESCRIPTION
**Pull request by sugi-chan**
read the docs hasnt been building and it looks like it the required files were removed previously... probably bc we had them in requirements.txt and in `extra_requires` in setup.py. but both got removed when we upgraded to using pip-compile based off of setup.py 

Pull Request Checklist
 - [ ] All tests in the `tests` folder pass with a local build
 - [x] Pull request includes a description of why we are doing this
 - [ ] Init files import new capabilities to appropriate level of package (if applicable)
 - [x] CHANGELOG has been updated
 - [x] Version in `_version.py` has been updated
 - [ ] README has been updated (if applicable)
 - [x] `requirements.txt` and `requirements-dev.txt` have been recompiled if requirements in `setup.py` or `requirements-dev.in` changed.

### CODE REVIEW

**Comment by sugi-chan** [comment]
if u want to test you can install the library and then make the docs something like:

```
pip install -e .

cd docs/

make html

```

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,17 +3,23 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project uses [Semantic Versioning](http://semver.org/).
 
-# [3.1.0] - 2020-02-03
+# [3.1.1] - 2021-02-03
+  ### Fixed
+  - sphinx and m2r were no longer in requirements files so docs failed added sphinx and had to update to m2r2 because m2r is not actively supported
+  ### Fixed
```
Comment (peterskipper) [inline CHANGELOG.md:9]: Think you can put `Fixed` once and add all the fixes underneath as bullet points with the dash

**Review by peterskipper — APPROVED** [review]
LGTM. Left a small suggestion for the CHANGELOG

File: CHANGELOG.md (hunk 1 above)
Comment (sugi-chan) [reply CHANGELOG.md:9]: that seems like a sane person thing to do

**Pull request merged**